.gitignore

```diff
@@ -0,0 +1,362 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
This is just the standard C# gitignore from:

https://github.com/github/gitignore/blob/master/VisualStudio.gitignore